challengeMk2/ViewModels/SystemDetailViewModel.cs

```diff
@@ -63,17 +63,17 @@ public SystemDetailViewModel(StarSystem selectedSystem = null)
             Title = "Star System Details";
             //Title = selectedSystem?.Name;
 
-            CurrentSystem = selectedSystem;
-
-          
+            CurrentSystem = selectedSystem;
+
+
             UpdateSystemData();
```

Comment:
Avoid calling an async method from a view model's constructor.

Reply:
Sorry, it is my fault, this class should be deleted. 
It is replaced by SystemDetailCarouselViewModel.cs and not be used anymore.